assets/javascripts/discourse/components/chat-channel-toggle-open-modal-inner.js

```diff
@@ -0,0 +1,58 @@
+import Component from "@ember/component";
+import { CHANNEL_STATUSES } from "discourse/plugins/discourse-chat/discourse/models/chat-channel";
+import I18n from "I18n";
+import discourseComputed from "discourse-common/utils/decorators";
+import { action } from "@ember/object";
+import { ajax } from "discourse/lib/ajax";
+import { inject as service } from "@ember/service";
+import { popupAjaxError } from "discourse/lib/ajax-error";
+
+export default Component.extend({
+  chat: service(),
+  tagName: "",
+  chatChannel: null,
+
+  @discourseComputed()
```

Comment:
Does this need to be a computed property when it isn't computing anyting?

Reply:
A philosophical question for the ages :) Will change to a simple `get X`